### PR TITLE
Setup Leaders for VSD

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/171194032

VSD is moving forward with categories more aligned with their Buyer's Guide.

Automation equipment
Cameras - visible
Cameras – 3D
Cameras – non-visible
Cameras – Specialty (High-speed, scientific)
Camera accessories
Image sensors
Lenses/optics
Lighting/illumination
Software
Image capture and system boards

Please place the Leaders block under the last row of content wells.

Screen Shot 2020-02-14 at 4 04 31 PM
Screen Shot 2020-02-14 at 4 04 15 PM

### CODE REVIEW

**Comment by Alixers505** [comment]
@brandonbk - could you take a look at this and let me know if I'm overlooking something? I've modeled it after your recent PRs.

File: packages/leaders/scss/index-page.scss [hunk 1]
```diff
@@ -33,3 +33,16 @@
     }
   }
 }
+
+.leaders {
```
Comment (zarathustra323) [inline packages/leaders/scss/index-page.scss]: Does this also get applied to the "non-rounded" sites, e.g. Security Info Watch?
If so, this should only be applied to the "Pennwell" sites, not everyone.
You would need to create a separate SCSS file for this and import it only into the sites that need it.

Comment (zarathustra323) [reply packages/leaders/scss/index-page.scss]: Or, is the rounding and box shadow disabled for those? I just need verification that by adding this it doesn't change the styling of the non-Pennwell sites.

Comment (Alixers505) [reply packages/leaders/scss/index-page.scss]: I wasn't aware that there was a different styling expectation for sites within the same repo. I'll look into a way to apply it to only specific sites, that wouldn't require replicating the css through each individual file.

Comment (zarathustra323) [reply packages/leaders/scss/index-page.scss]: Can you update the primary screenshots and also provide a screen for SIW? Thanks!

**Comment by Alixers505** [comment]
VSD - Rounded/boxed
Screen Shot 2020-02-17 at 11 43 13 AM

SIW - Not rounded/boxed
Screen Shot 2020-02-17 at 11 56 00 AM

**Pull request closed without merging**